### DISCUSSION TITLE
Supabaseでチャットアプリを作る ハンズオン、できたよスタンプ

### DISCUSSION THREAD

**Discussion by jiyuujin** [flutter-osaka-dev/flutter-chat#3] [Show and tell]
できたら、スタンプを押してみてね

**Comment by jiyuujin** [comment]
[Supabase のプロジェクト作成](https://zenn.dev/dshukertjr/books/flutter-supabase-chat/viewer/page2)
できたひと

**Comment by jiyuujin** [comment]
[Flutter create & Flutter と Supabase の紐付け・準備](https://zenn.dev/dshukertjr/books/flutter-supabase-chat/viewer/page3)
できたひと

**Comment by jiyuujin** [comment]
[登録・ログインページ](https://zenn.dev/dshukertjr/books/flutter-supabase-chat/viewer/page4)
できたひと

**Comment by jiyuujin** [comment]
[チャットのロジック](https://zenn.dev/dshukertjr/books/flutter-supabase-chat/viewer/page5)
できたひと

**Comment by jiyuujin** [comment]
[UI仕上げ](https://zenn.dev/dshukertjr/books/flutter-supabase-chat/viewer/page6)
できたひと

**Reply by dshukertjr** [reply]
supabaseUrl: `https://sqqhskwlxzqwzmdpmzre.supabase.co`
anonKey: `eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InNxcWhza3dseHpxd3ptZHBtenJlIiwicm9sZSI6ImFub24iLCJpYXQiOjE2ODE5ODU1MjAsImV4cCI6MTk5NzU2MTUyMH0.Nizhj8LkDav8QtvRUeDD-iZJxqCBsFaelPDjSgllslw`
